getstarted.md

```diff
@@ -22,6 +22,70 @@ with Hadoop, as well as extensions to that foundation that make for totally
 awesome demos. You'll also find a few videos related to our work and of course,
 how to contact us!
 
+First, though, let's just take a look at how straightforward it is to get started with very complex software. You can do it during a lunch break:
+
+## A Lunch Break Project -- Visualize Earthquake Data with Zeppelin
+
+*This tutorial assumes that you have Juju 2.0 installed and bootstrapped. If you don't, try installing [the devel version of charmbox](https://github.com/juju-solutions/charmbox/tree/devel), setup a free cloud account via our [developer program](https://developer.juju.solutions), and follow the [Juju getting started guide](https://jujucharms.com/docs/devel/getting-started).*
+
+1. Deploy the hadoop processing bundle.
+```
+juju deploy hadoop-processing
```

Comment:
does the earthquake demo require hadoop?  if the notebook stores stuff in hdfs, then the answer is no and you can ignore me, but if it doesn't, perhaps `spark-processing` would be a better fit?  in that case, we could drop the `plugin` talk below.